### PR TITLE
Capture errors from Conan and log them to output window and log file

### LINKED ISSUE #118: Better error messaging for
When something fails, we get a 

```
Conan has returned exit code '1' while processing configuration
'Architecture... Please check file '...\.conan\conan_7101ceb0-d
ba-457c-bcb5-b9595a8619bb.log' for details
```

but nothing interesting is in that file 😖. At least we need to copy there all the output from the Conan execution.

### PR DESCRIPTION
**Pull request by jgsogo**
Capture the errors from Conan output and log them to all available windows.

It also modifies the return type from ConanRunner, probably it is better to return just the `ProcessStartInfo` and let the consumer decide to run the command synchronously or not.

closes #118

### CODE REVIEW

File: Conan.VisualStudio/Services/ConanService.cs [hunk 1]
```diff
@@ -117,43 +118,50 @@ private async Task InstallDependenciesAsync(ConanRunner conan, ConanProject proj
...
                     await logStream.WriteLineAsync(message);
 
-                    using (var reader = process.StandardOutput)
+                    using (Process exeProcess = Process.Start(process))
                     {
-                        string line;
-                        while ((line = await reader.ReadLineAsync()) != null)
+                        int exitCode = await exeProcess.WaitForExitAsync();
+
+                        string output = await exeProcess.StandardOutput.ReadToEndAsync();
```
Comment (SSE4) [inline Conan.VisualStudio/Services/ConanService.cs]: so, this way we have all stdout, then all stderr, which might be hard to make a correlation, is it possible to capture them in order?

Comment (jgsogo) [reply Conan.VisualStudio/Services/ConanService.cs]: Yes, we should tell the process to redirect `stderr` to `stdout` (o vice-versa). It is outside of my C# knowledge right now 😆 

Probably we need a helper function to call any `ProcessStartInfo` and do the right thing with all of this. Do you think this is needed for this PR or just open a new issue to fix it?

Comment (SSE4) [reply Conan.VisualStudio/Services/ConanService.cs]: let's do it now, as we're touching that code, let's make it correct at one shot.
shouldn't be too hard, something like that could be used as a base:
https://gist.github.com/rivy/030df34899e4305ea5837586bf1ff7a4

**Comment by jgsogo** [comment]
Nice improve over my code 😊

**Pull request merged**